src/main/java/co/cask/plugin/format/MultiTableDBInputFormat.java

```diff
@@ -59,15 +59,19 @@
 
     try (Connection connection = dbConf.getConnection()) {
       DatabaseMetaData dbMeta = connection.getMetaData();
-      ResultSet tables = dbMeta.getTables(null, null, dbConf.getTableNamePattern(), new String[] {"TABLE"});
+      ResultSet tables = dbMeta.getTables(null, dbConf.getSchemaName(), null, new String[] {"TABLE"});
```

Comment:
we should still support a table name pattern, but also use the schema pattern.

Reply:
Updated.